test/nettests/web_connectivity.cpp

```diff
@@ -16,86 +16,6 @@ TEST_CASE("Synchronous web connectivity test") {
                                                    test::nettests::run_test);
 }
 
-TEST_CASE("Make sure that IP address scrubbing works") {
```

Comment:
Test transformed into an unittest. Too much flapping and weird issues in depending from external services. The goal is to see if we scrub the address when requested. To do so, it suffices to see if we scrub it. No need to use the net.